### ISSUE TITLE
[Installer] Possibility to skip database connection setup (database.yml)

### ISSUE THREAD

**Issue by Crell** [pimcore/pimcore#6638]
# Bug Report

I am trying to update the installer for Pimcore on Platform.sh, and running into an issue.  Specifically, even though an `/appconfig/installer.yml` file is included, the installer is tryign to write an `/app/config/local/database.yml` file.  I want it to skip that entirely, because the DB information will always be provided via environment variables and injected into the container through a custom parameters file.

I don't believe this problem existed on earlier versions, but with the latest Pimcore release I'm running into this wall and haven't figured out a way around it.

### Expected/Desired behavior

Installer detects that DB information is already available in the container and just keeps on going without trying to write any YAML files to disk.

### Actual behavior

The installer fails with:

```
 [ERROR] The following errors were encountered during installation                                                      
                                                                                                                        

 * Warning: file_put_contents(/app/app/config/local/database.yml): failed to open stream: Read-only file system
```

Unfortunately I think this hard-blocks the Pimcore template on Platform.sh for the time being.

**Comment by brusch** [comment]
I guess we need another CLI option to skip the database connection setup ...

**Comment by Crell** [comment]
In my experience (I've worked with a ton of installers in this job), the best alternative for the system to accept environment variable-based configuration for services and other environment-specific stuff, then have the installer automatically skip asking for anything it can already find via env vars.  (A flag on a CLI installer to skip asking is also acceptable, but less user-friendly.)  As long as the env vars you need are documented, it's then reasonably straightforward for us to write the code to map the hosting provider's values to the env vars the application needs.

The big thing is to make it straightforward and well-documented to be able to install without writing config files to disk, because that's the death-knell for applications on cloud providers.

**Comment by dvesh3** [comment]
Hi @Crell, can you please check if #6897 solves your issue? thanks!

**Comment by chadwcarlson** [comment]
@dvesh3 Following up on @Crell 's work here. Is `--skip-database-config` supposed to be accessible from skeleton 2.7.0? I'm getting an error with `pimcore-install` that `--skip-database-config` is not an available  option.

**Comment by dvesh3** [comment]
@chadwcarlson we merged the PR on master and `--skip-database-config` option will be available in pimcore/pimcore v6.8.0. Skeleton 2.7.0 is using latest version 6.7.0.

**Comment by jdreesen** [comment]
It will be released with Pimcore v6.8, see: #6897

**Comment by chadwcarlson** [comment]
Got it, thank you both.
